promgen/templates/promgen/project_detail_hosts.html

```diff
@@ -48,7 +48,7 @@
 {% if project.farm %}
   {% if project.farm.editable %}
     <a href="{% url 'hosts-add' project.farm.id %}" class="btn btn-primary">
-      {% trans "Add new host" %}
+      {% trans "Register hosts" %}
```

Comment:
I wonder if the casing of this, needs to match above "Hosts" vs "hosts"

Reply:
You are right, `Register Hosts` looks better and more consistent with all the other buttons where the first letter of each work is capitalized: `Register Farm`, `Register Project`, `Register Service`, etc.

I'll do the changes.

Reply:
Fixed in:
- c238e0a: properly capitalize the words and make them match with the ID
- f61af71: add translation for a "Register Hosts" button that I missed 
- a700cd3: recompile messages